add metrics content (#5224)
getsentry/sentry-docs: src/docs/product/sentry-basics/events-metrics/index.mdx

---
title: Metrics
sidebar_order: 25

> description: "Learn more about the different ways Sentry treats your events."

Author: ```suggestion
description: "Learn more about how Sentry uses your event data to generate metrics."
```

---

> 

Author: ```suggestion
<Note>

This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in joining this limited release, please reach out at [abc@sentry.io](mailto:abc@sentry.io).

</Note>

```

> When you're using sentry.io, you are seeing information that's generated by one of two types of events:

Author: ```suggestion
When you're using [sentry.io](https://sentry.io), you are seeing information that's generated by one of two types of events:
```

> - processed events - All the events that you send to Sentry, based on your SDK sample rate. Underlies _metrics-based_ data.

Author: ```suggestion
- processed events - All the events that you send to Sentry, based on your SDK sample rate. 
```

> - processed events - All the events that you send to Sentry, based on your SDK sample rate. Underlies _metrics-based_ data.
> - sampled events - A sub-set of your processed events, based on your dynamic sampling setings. Underlies _events-based_ data.

Author: ```suggestion
- Processed events - All the events that you send to Sentry, based on your SDK sample rate. Underlies _metric-backed_ data.
- Indexed events - A sub-set of your processed events, based on your dynamic sampling settings. Underlies _events-based_ data.
```

> _Processed_ events can include all the events that occur in your application or a sub-set. This is based on the sample rate that you set in the SDK. These events represent aggregate data. This is used in areas of the application like **Performance** and **Dashboards**. The larger your set of processed events is large enough, the better able we are to provide you with statisically significant insights into the performance of your application. In sentry.io, you may see the data generated from these events referred to as _metrics_ or _metrics-based_.

Author: ```suggestion
_Processed_ events can include all the events that occur in your application or a sub-set of those. The number of these events that you send to Sentry is based on the sample rate that you set in the SDK. These events represent aggregate data. This is used in areas of the application like **Performance** and **Dashboards**. The larger your set of processed events, the better able we are to provide you with statistically significant insights into the performance of your application. In [sentry.io](https://sentry.io), you may see the data generated from these events referred to as _metrics_ or _metric-backed_.
```

Because these events represent aggregate data, when your viewing information in sentry.io that's based on these events, or based on metrics data, you can't drill down into the specifics of these events.

> _Sampled_ events are a sub-set of your processed events, and this sub-set is created based on your dynamic sampling settings. For these events, all of the event data is available to you, so you can drill down into each event for further insights. In sentry.io, you may see the data generated from these events referred to as _transaction data_ or _events-based_.

Author: ```suggestion
_Sampled_ events are a sub-set of your processed events, and this sub-set is created based on your dynamic sampling settings. For these events, all of the event data is available to you, so you can drill down into each event for further insights. In sentry.io, you may see the data generated from these events referred to as _transaction data_ or _events-based_. The events that you decide to keep from your processed events that Sentry stores for you. Currently, this only applies to transaction events. For these events, we store the complete transaction data rather than just aggregate data.
```

> ![Image of a breakdown of event types](metrics-events-subsets.png)

Author: ```suggestion
Metrics are generated from all of your processed events, or all the events that you send to Sentry.

![Image of a breakdown of event types](metrics-events-subsets.png)
```

<!-- TO DO: TALK ABOUT DIFFERENT USE CASES HERE -->

## Custom Performance Measurements

> TBA

Author: ```suggestion
Custom performance metrics for transactions allow you to define metrics that are important to your application and send them to Sentry. Currently you can set these for the [JavaScript SDK](/platforms/javascript/performance/instrumentation/performance-metrics/#custom-metrics).

```
